src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/ExpressionCompilerTests.vb

```diff
@@ -2082,6 +2082,43 @@ End Class"
 }")
         End Sub
 
+        ''' <summary>
+        ''' Generate PrivateImplementationDetails class
+        ''' for initializer expressions.
+        ''' </summary>
+        <Fact>
+        Public Sub ExpressionInitializerExpression()
+            Const source =
+"Class C
+    Shared Sub M()
+    End Sub
+End Class"
+            Dim compilation0 = CreateCompilationWithMscorlib(
+                {source},
+                references:={SystemCoreRef},
+                compOptions:=TestOptions.DebugDll.WithModuleName("MODULE"),
+                assemblyName:=ExpressionCompilerUtilities.GenerateUniqueName())
+            Dim runtime = CreateRuntimeInstance(compilation0)
+            Dim context = CreateMethodContext(
+                runtime,
+                methodName:="C.M")
+            Dim resultProperties As ResultProperties = Nothing
+            Dim errorMessage As String = Nothing
+            Dim testData = New CompilationTestData()
+            context.CompileExpression("{ 1, 2, 3, 4, 5 }", resultProperties, errorMessage, testData)
```

Comment:
What's the type of this expression?  Would it be interesting to test the return type of the method?